Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection_test.py

```diff
@@ -2178,3 +2195,65 @@ def test_build_common_files_query(self):
             )
             actual = cu.build_common_files_query()
             assert actual == expected
+
+
+def test_get_machine_users_command(mocker):
+    mocker.patch.object(client_mocker, 'get_machine_users', return_value=MACHINE_USER_DATA)
+    results = get_machine_users_command(client_mocker, {'machine_id': "123abc"})
+    assert results.outputs == [{
+        "AccountName": "user1",
+        "AccountDomain": "contoso",
+        'AccountSID': None,
+        "DomainAdmin": True,
+        "FirstSeen": "2019-12-18T08:02:54Z",
+        "ID": "contoso\\user1",
+        "LastSeen": "2020-01-06T08:01:48Z",
+        'LeastPrevalentMachineID': None,
+        'LogonCount': None,
+        "LogonTypes": "Interactive",
+        'MachineID': '123abc',
+        'MostPrevalentMachineID': None,
+        "NetworkUser": False,
+    }]
+
+
+def test_get_machine_alerts_command(mocker):
```

Comment:
Please add a “Given, When, Then” to the test, example can be found in the other tests in the file

Reply:
Added given, when then